lib/fog/aws/models/compute/vpc.rb

```diff
@@ -10,8 +10,11 @@ class VPC < Fog::Model
         attribute :tags,             :aliases => 'tagSet'
         attribute :tenancy,          :aliases => 'instanceTenancy'
         attribute :is_default,       :aliases => 'isDefault'
-        attribute :ipv_6_cidr_block_association_set,  :aliases => 'ipv6CidrBlockAssociationSet'
-        attribute :amazon_provided_ipv_6_cidr_block,  :aliases => 'amazonProvidedIpv6CidrBlock'
+
+        attribute :cidr_block_association_set, :aliases => 'cidrBlockAssociationSet'
+
+        attribute :ipv_6_cidr_block_association_set, :aliases => 'ipv6CidrBlockAssociationSet'
```

Comment:
I guess it was already there, but it seems surprising to me that this is `ipv_6_cidr_block_association_set` instead of `ipv6_cidr_block_association_set`. Does that make sense? I just happened to notice here (and have the same question about the new attribute below). What do you think?

Reply:
I think you are right. We could just add alias for backward compatibility (ipv_6_cidr_block_association_set is already released...)

Reply:
I hadn't even thought about that, but that sounds good. That way we can have more consistent names, but avoid breaking anybody. Sound good?